packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -3766,6 +3766,10 @@
         "properties": {
           "interval": {
             "type": "keyword"
+          },
+          "rrule": {
```

Comment:
probably don't need it at all, unless we're doing an `exists` query in the code somewhere currently?

Reply:
removed all